web/client/codechecker_client/cmd/store.py

```diff
@@ -542,23 +559,62 @@ def parse_report_files(report_files):
     changed_files = set()
     missing_source_files = set()
 
-    with concurrent.futures.ProcessPoolExecutor() as executor:
-        for report_f, v in zip(report_files,
-                               executor.map(parse_collect_plist_info,
-                                            report_files)):
-
-            report_file_info, source_in_reports = v
-
-            if report_file_info.store_it:
-                files_to_compress.add(report_f)
-
-            source_file_info.update(source_in_reports.source_info)
-            changed_files = \
-                changed_files | source_in_reports.changed_since_report_gen
-            main_report_positions.extend(
-                report_file_info.main_report_positions)
-            missing_source_files = \
-                missing_source_files | source_in_reports.missing
+    def _populate_data(report_f, v, files_to_compress, source_file_info, main_report_positions, changed_files, missing_source_files):
+        """updates bunch of interal sets/lists with given report data, intended for loop below"""
+        report_file_info, source_in_reports = v
+
+        if report_file_info.store_it:
+            files_to_compress.add(report_f)
+
+        source_file_info.update(source_in_reports.source_info)
+        changed_files = \
+            changed_files | source_in_reports.changed_since_report_gen
+        main_report_positions.extend(
+            report_file_info.main_report_positions)
+        missing_source_files = \
+            missing_source_files | source_in_reports.missing
+
+        return (files_to_compress,
+                source_file_info,
+                main_report_positions,
+                changed_files,
+                missing_source_files)
+
+    # Currently ProcessPoolExecutor fails completely in windows
+    # Reason is most likely combination of venv and fork() not
+    # being present in windows, so stuff like setting up
+    # PYTHONPATH in parent CodeChecker before store is executed
+    # are lost. Maybe
+    if util.is_windows():
```

Comment:
We can use the similar approach what I mentioned above:
```py
def parse_report_files(report_files, zip_iter=map):
    """Parse and collect source code information mentioned in a report file.

    Collect any mentioned source files wich are missing or changed
    since the report generation. If there are missing or changed files
    the report will not be stored.
    """

    files_to_compress = set()
    source_file_info = {}
    main_report_positions = []
    changed_files = set()
    missing_source_files = set()

    for report_f, v in zip(report_files,
                           zip_iter(parse_collect_plist_info,
                                    report_files)):

        report_file_info, source_in_reports = v

        if report_file_info.store_it:
            files_to_compress.add(report_f)

        source_file_info.update(source_in_reports.source_info)
        changed_files = \
            changed_files | source_in_reports.changed_since_report_gen
        main_report_positions.extend(
            report_file_info.main_report_positions)
        missing_source_files = \
            missing_source_files | source_in_reports.missing

    return (source_file_info,
            main_report_positions,
            files_to_compress,
            changed_files,
            missing_source_files)
```
On the call side:
```py
    if sys.platform == "win32":
        (source_file_info,
         main_report_positions,
         files_to_compress,
         changed_files,
         missing_source_files) = parse_report_files(report_files)
    else:
        with concurrent.futures.ProcessPoolExecutor() as executor:
            (source_file_info,
             main_report_positions,
             files_to_compress,
             changed_files,
             missing_source_files) = parse_report_files(report_files,
                                                        executor.map)
```